xbmc/addons/AddonInstaller.cpp

```diff
@@ -231,8 +231,8 @@ std::vector<std::string> CAddonInstaller::RemoveOrphanedDepsRecursively() const
       }
       else
       {
-        CLog::Log(LOGERROR, "CAddonMgr::{}: failed to remove orphaned dependency: {}", __FUNCTION__,
-                  dep->Name());
+        CLog::Log(LOGERROR, "CAddonMgr::{}: failed to remove orphaned add-on/dependency: {}",
+                  __FUNCTION__, dep->Name());
```

Comment:
I think __func__ is now preferred. I don’t recall why, @AlwinEsch knows!

Reply:
A background is the C ++, this is with him and "__FUNCTION__" with "C". The other is that Windows should no longer add its class name, but would have to look again whether it really fits now.

Before it would come on windows as "CAddonMgr::CAddonMgr::RemoveOrphanedDepsRecursively", with two time the class name.

Reply:
docs say microsoft would not throw out the class
https://docs.microsoft.com/de-de/cpp/cpp/func?view=msvc-160

and so wouldn‘t gnu/clang  - i think it‘s ok this way

Reply:
Oops, disregard. I didn't see that `__func__` was the solution.